From b137f1ff7179aabb8c9fea225a0e17c1ab8ad629 Mon Sep 17 00:00:00 2001
From: Dan Jacques <dnj@chromium.org>
Date: Wed, 14 Jun 2017 08:36:47 -0700
Subject: [PATCH 2/2] Infra-specific extensions.

---
 Makefile   | 6 ++++++
 exec_cmd.c | 9 +++++++++
 2 files changed, 15 insertions(+)

diff --git a/Makefile b/Makefile
index ccfd5d55d..89a2737f3 100644
--- a/Makefile
+++ b/Makefile
@@ -1693,6 +1693,12 @@ ifdef NO_INSTALL_HARDLINKS
 	export NO_INSTALL_HARDLINKS
 endif
 
+ifndef NO_PERL
+ifdef PERL_LIB_PATH
+	BASIC_CFLAGS += -DGIT_PERL_LIB_PATH="\"$(PERL_LIB_PATH)\""
+endif
+endif
+
 ### profile feedback build
 #
 
diff --git a/exec_cmd.c b/exec_cmd.c
index f745fcbf9..a7484ad7e 100644
--- a/exec_cmd.c
+++ b/exec_cmd.c
@@ -161,6 +161,15 @@ void git_set_exec_path(const char *exec_path)
 	 * Propagate this setting to external programs.
 	 */
 	setenv(EXEC_PATH_ENVIRONMENT, exec_path, 1);
+
+  /** Infra: Propagate values to scripts. */
+#if defined(RUNTIME_PREFIX) && defined(GIT_PERL_LIB_PATH)
+  setenv("GITPERLLIB", system_path(GIT_PERL_LIB_PATH), 1);
+#endif
+#if defined(RUNTIME_PREFIX)
+  setenv("GIT_TEXTDOMAINDIR", system_path(GIT_LOCALE_PATH), 1);
+#endif
+  /** END Infra */
 }
 
 
-- 
2.14.1.chromium11

