--- a/go/src/appengine_internal/api_dev.go 2015-02-12 04:38:22.000000000 -0800
+++ b/go/src/appengine_internal/api_dev.go  2015-03-26 18:52:53.902590265 -0700
@@ -57,7 +57,6 @@
 	// Read configuration from stdin when the application is being run by
 	// devappserver2. The user application should not be reading from stdin.
 	if os.Getenv("RUN_WITH_DEVAPPSERVER") != "1" {
-		log.Print("appengine: not running under devappserver2; using some default configuration")
 		return
 	}
 	c := readConfig(os.Stdin)
